### ISSUE TITLE
Sub-Folders Not Being Picked Up

### ISSUE THREAD

**Issue by JimmyTime** [sunnyark/civitai-shortcut#11]
Is there a limit to how many levels deep the scan will go? I seem to be able to have my already folder-sorted LORAs picked up only if they are 1 sub-folder deep, not 2-4 like I have mine.

**Comment by sunnyark** [comment]
All models assume that only up to type_base_folder (e.g. lora) / model_namefolder / version_folder are used. The scan in manage->scan and update searches up to the subfolders, but any additional folders created by extensions are only managed up to model_namefolder / version_folder.

**Comment by micky2be** [comment]
Same here, I already have a classification system using folders.
Can you at least offer a way t scan existing models?

**Comment by sunnyark** [comment]
@micky2be Have you tried using the 'Scan models for Civitai' feature under Assistance->Scan and Update Models? This feature consists of two steps.

First, it performs the 'Scan Models.' This step searches for model files in your possession that do not have civitai information files. If there are models that meet the criteria, a list of files will appear.

Second, from the list, you can select the models you want to register along with the desired options, and then perform 'Create Model Information.' This step generates a hash value for the model and compares it with the files existing in Civitai to create civitai information. Depending on your chosen options, you can register it as a shortcut.

The options 'Create a model folder corresponding to the model type' and 'Create individual model version folder' move the model files to the folder storage system of the extension you are using. If you are already organizing your folders independently, please do not check these options.

**Comment by micky2be** [comment]
@sunnyark RIGHT! "**Scan downloaded models for shortcut registration**" worked.
It did ignore some old checkpoints not available anymore on Civitai though.

Thanks for your work
